dashboard/src/actions/apps.ts

```diff
@@ -244,7 +244,7 @@ export function fetchAppsWithUpdateInfo(
       dispatch(
         getAppUpdateInfo(
           cluster,
-          namespace,
+          app.namespace,
```

Comment:
A couple of things confuse the code here, imo:


* the `all` function argument is actually `allStatuses` not all apps across namespaces - can we rename it to be explicit (it's always passed as true from the only call-site afaict, so we could even get rid of it).
* the `fetchApps` and subsequent `App.listApps` have functionality to support fetching apps across all namespaces which is the cause of this change, but it'd be a lot easier to reason about this code if it was obvious why `namespace` can't be used here (because it could have the value `_all`. Perhaps a comment, or renaming it `namespace_or_all` or something.

Reply:
> * the `all` function argument is actually `allStatuses` not all apps across namespaces - can we rename it to be explicit (it's always passed as true from the only call-site afaict, so we could even get rid of it).

Yes, since we have deleted the code for the Hex UI we can delete it.

> * the `fetchApps` and subsequent `App.listApps` have functionality to support fetching apps across all namespaces which is the cause of this change, but it'd be a lot easier to reason about this code if it was obvious why `namespace` can't be used here (because it could have the value `_all`. Perhaps a comment, or renaming it `namespace_or_all` or something.

:+1: